client/blocks/plan-thank-you-card/docs/example.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import sitesList from 'lib/sites-list';
+import PlanThankYouCard from 'blocks/plan-thank-you-card';
+
+const sites = sitesList();
+
+const PlanThankYouCardExample = React.createClass( {
+	displayName: 'PlanThankYouCard',
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/blocks/plan-thank-you-card">Plan Thank You Card</a>
+				</h2>
+				<PlanThankYouCard selectedSite={ sites.getPrimary() } />
```

Comment:
`sitesList()` are an outdated structure and we prefer not to use them when not necessary. In order to get a selected site, could you please use `getSelectedSite` selector from `client/state/ui/selectors`? You'll also need to connect this component to Redux store. See `blocks/post-item/docs/example` as an example.